scripts/testServer.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+PS1="$"
+basedir="$(pwd -P)"
```

Comment:
I would prefer we keep the "you give the basedir to the script as an argument" system consistent throughout all the scripts, so that if we restructure again later it will be simpler to modify everything.